File: dataset/output/OPENAI_gpt-4o/Task25_OPENAI_gpt-4o.c
#include <stdio.h>
#include <string.h>

void number(char lines[][100], int size) {
    for (int i = 0; i < size; i++) {
        printf("%d: %s
", i + 1, lines[i]);
    }
}

int main() {
    char lines[][100] = {"a", "b", "c"};
    int size = sizeof(lines)/sizeof(lines[0]);
    number(lines, size);
    return 0;
}

File: dataset/output/OPENAI_gpt-4o/Task25_OPENAI_gpt-4o.java
import java.util.List;
import java.util.ArrayList;

public class LineNumbering {
    public static List<String> number(List<String> lines) {
        List<String> result = new ArrayList<>();
        for (int i = 0; i < lines.size(); i++) {
            result.add((i + 1) + ": " + lines.get(i));
        }
        return result;
    }

    public static void main(String[] args) {
        List<String> lines = List.of("a", "b", "c");
        System.out.println(number(lines));
    }
}